common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -125,10 +125,13 @@ public static void load(String originalName, ClassLoader loader) {
             }
         }
 
-        if (url == null) {
-            // Fall back to normal loading of JNI stuff
+        try {
+            // first try to load from java.library.path
             loadLibrary(loader, name, false);
+            logger.info("{} was loaded from java.libary.path", name);
             return;
+        } catch (Exception ex) {
+            logger.debug("{} cannot be loaded from java.libary.path", name, ex);
```

Comment:
@eun-ice yes... another idea (and maybe even the better) would be to try the workdir first and only if it fails try java.library.path. This way the old "order" is maintained. WDYT ?